### PR TITLE
React metrics capstone project - Currency Exchange rates metrics

### PR DESCRIPTION
**Pull request by prius2055**
### React metrics capstone project - Currency Exchange rates metrics

In this project;

- I have built a currency exchange application.
- The application shows a list of some world currencies.
- The user can search through a list of currencies using a drop-down
- The user can compare the exchange rates of any given currency with the other currencies.
- The design was implemented using the API from [this link](https://www.vatcomply.com/documentation)

For the project documentation, click [here](https://www.loom.com/share/97429e1f5ebf4b1da8d884c218a9539a)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,179 @@
-# react-metrics-app
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
```
Comment (bobb-Rob) [inline README.md:4]: - [ ] The [video presentation](https://www.loom.com/share/5bee8d89db6343599ccb7c8192f51ee5) lacks highlights of a piece of cod that you finds interesting or are proud of making. Kindly redo this video including a show of a piece of code that you are proud of.

File: README.md [hunk 2]
```diff
@@ -1 +1,179 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse team for their support.
+
+I would also like to appreciate the creative commons team for the graphical design and inspiration
+
+> Click on this link to checkout them out.
+
+[Creative commons] (https://creativecommons.org/licenses/by-nc/4.0/)
```
Comment (bobb-Rob) [inline README.md]: - [ ] You did well mentioning Creative Commons, but the actual designer is [Nelson Sakwa](https://www.behance.net/sakwadesignstudio). Please explicitly mention him and give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @prius2055, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (prius2055) [reply README.md]: Thanks, this has been resolved

File: README.md (hunk 1 above)
Comment (prius2055) [reply README.md:4]: Yes, I have uploaded another video. Thank you

File: README.md
Comment (KanzaTahreem) [inline README.md:1]: - [ ] Great job mentioning the feature, however, as suggested earlier, kindly highlight of **_a piece of code_** that you finds interesting or are proud of making. You can highlight it from your editor 🎥📌Let me know if you have any question 😄

File: README.md [hunk 3]
```diff
@@ -1 +1,179 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse team for their support.
+
+I would also like to appreciate the creative commons team for the graphical design and particularly Nelson Sakwa the author of the design.
```
Comment (KanzaTahreem) [inline README.md]: - [ ] Please note that the creative commons is an organization that provides free licences for creators to use. It is not someone who provides you designs. The actual designer is [Nelson Sakwa](https://www.behance.net/sakwadesignstudio). Kindly make sure to give appropriate credits to the designer as suggested earlier ⛏🤓, and you can mention creative commons like the following. Please feel free to use it 👇

  I would like to thank [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @prius2055 🙋‍♀️

Good job so far! I really appreciate the effort you put in!  **_However, there are few things that still need to be fixed. But you are almost there!_** 💪 🏁

so_close

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Well done with the design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (prius2055) [reply README.md:1]: Done

File: README.md (hunk 3 above)
Comment (prius2055) [reply README.md]: Fixed

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @prius2055 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**